Adapter/PhpcrOdmAdapter.php

```diff
@@ -137,12 +138,24 @@ public function createAutoRoute(UriContext $uriContext, $contentDocument, $autoR
             $parentDocument = $document;
         }
 
+        $finalAutoRoutePath = $path . '/' . $headName;
+        $node = $this->dm->find(null, $finalAutoRoutePath);
```

Comment:
A `$document` variable already exists in this scope and is used below. What about `$autoRouteDocument` as new variable name?